### PR TITLE
JP Onboarding: Move Summary Completed Steps to Separate Component

### PR DESCRIPTION
**Pull request by ockham**
To test: Verify that JPO's Summary Page's Completed Links still work.

In a subsequent step, we will move both this file and `summary-todo-steps` one folder up.

### CODE REVIEW

File: client/jetpack-onboarding/steps/summary-completed-steps.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
```
Comment (tyxla) [inline client/jetpack-onboarding/steps/summary-completed-steps.jsx]: ABC police is watching you 🚔  😆

File: client/jetpack-onboarding/steps/summary-completed-steps.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import { get, map, without } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'gridicons';
+import Spinner from 'components/spinner';
+import {
+	getJetpackOnboardingPendingSteps,
```
Comment (tyxla) [inline client/jetpack-onboarding/steps/summary-completed-steps.jsx]: 🔤

File: client/jetpack-onboarding/steps/summary-completed-steps.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
+/** @format */
```
Comment (tyxla) [inline client/jetpack-onboarding/steps/summary-completed-steps.jsx]: Just noting that I'd also prefer if this file lived in the root directory of JPO (with regards to the PR description).

File: client/jetpack-onboarding/steps/summary-completed-steps.jsx (hunk 2 above)
Comment (ockham) [reply client/jetpack-onboarding/steps/summary-completed-steps.jsx]: I thought we did default imports before named ones? :thinking:

File: client/jetpack-onboarding/steps/summary-completed-steps.jsx (hunk 3 above)
Comment (ockham) [reply client/jetpack-onboarding/steps/summary-completed-steps.jsx]: See PR desc 😛 

> In a subsequent step, we will move both this file and `summary-todo-steps` one folder up.

It's easier to do it in this order: Once both components live in separate files, I can move them one level up _and_ change their `className`s at the same time.

File: client/jetpack-onboarding/steps/summary-completed-steps.jsx (hunk 2 above)
Comment (tyxla) [reply client/jetpack-onboarding/steps/summary-completed-steps.jsx]: I meant 🔤  for the order of the selectors here.

Comment (ockham) [reply client/jetpack-onboarding/steps/summary-completed-steps.jsx]: 😱 🙀

**Pull request merged**